apps/test/unit/lib/levelbuilder/rubrics/LearningGoalItemTest.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+import {expect} from '../../../../util/reconfiguredChai';
+import LearningGoalItem from '@cdo/apps/lib/levelbuilder/rubrics/LearningGoalItem';
+import sinon from 'sinon';
+
+describe('LearningGoalItem', () => {
+  let defaultProps;
+  let deleteItemSpy = sinon.spy();
+
+  beforeEach(() => {
+    defaultProps = {
+      deleteItem: deleteItemSpy,
+    };
+  });
+
+  it('renders correctly', () => {
+    const wrapper = mount(<LearningGoalItem {...defaultProps} />);
```

Comment:
It's generally preferable to use `shallow` when possible so I'm wondering if you could check that `EvidenceDescriptions` is rendered then move checking what it renders into its own test